setup.py

```diff
@@ -33,7 +33,7 @@
     install_requires=[
         "aioredis~=1.0",
         "msgpack~=0.6.0",
-        "asgiref~=2.1",
+        "asgiref~=3.0",
         "channels~=2.0",
```

Comment:
channels should be bumped as well, because channels >=2.0,<2.1.8 does not support asgiref ~=3.0.
I cannot bump this yet as I don't know the version of channel's next release.